projects/storefrontlib/cms-components/misc/icon/icon.component.ts

```diff
@@ -47,87 +47,100 @@ export class IconComponent {
    * The cxIcon directive is bound to the icon type. You can feed the `ICON_TYPE` to
    * accomplish a configurable button in the UI.
    */
-  @Input() set cxIcon(type: ICON_TYPE) {
+  @Input()
+  public set cxIcon(type: ICON_TYPE_STRING) {
```

Comment:
I think that as a convention we don't declare public members explicitely.

Reply:
ok, I'll fix that. I've seen both in the old code. and the linter didn't seem to complain